xds/src/test/java/io/grpc/xds/XdsCommsTest.java

```diff
@@ -284,8 +290,11 @@ public void standardMode_sendEdsRequest_getEdsResponse_withNoDrop() {
         .build();
     responseWriter.onNext(edsResponse);
 
-    verify(localityStore, times(2)).updateDropPercentage(ImmutableList.<DropOverload>of());
-    verify(localityStore, times(2)).updateLocalityStore(localityEndpointsMappingCaptor.capture());
+    verify(adsStreamCallback, times(1)).onWorking();
+    verify(adsStreamCallback, never()).onAllDrop();
+    verify(adsStreamCallback, never()).onError();
```

Comment:
Can the above two lines be replaced by `verifyNoMoreInteractions()`? (Ditto for the same cases below)